Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "bms-rs"
-version = "0.4.1"
+version = "0.4.2"
```

Comment:
Please don't bump the version manually, because it will be updated automatically with CI.